symforce/codegen/backends/python/python_config.py

```diff
@@ -56,3 +54,9 @@ def printer(self) -> CodePrinter:
         from symforce.codegen.backends.python import python_code_printer
 
         return python_code_printer.PythonCodePrinter()
+
+    @staticmethod
+    def format_matrix_accessor(key: str, i: int, j: int, *, shape: T.Tuple[int, int]) -> str:
+        if shape[1] == 1:
+            return f"{key}[{i}]"
```

Comment:
Maybe assert that j is 0 here?

Reply:
Should we instead just always assert that `0 <= i < shape[0]` and `0 <= j < shape[1]`?

Reply:
Yeah that seems reasonable too